x-pack/filebeat/docs/inputs/input-aws-cloudwatch.asciidoc

```diff
@@ -40,16 +40,24 @@ The `aws-cloudwatch` input supports the following configuration options plus the
 [float]
 ==== `log_group_arn`
 ARN of the log group to collect logs from.
+The ARN may refer to a log group in a linked source account.
```

Comment:
Can we also make a note here to point out: "If the log group is in a source account and you are using a monitoring account, you must use the log group ARN."

Reply:
Done, please see the commit - https://github.com/elastic/beats/pull/41188/commits/e280d23c0441465e1de27c9dd11e8e2db5eeb11a